inspec.yml

```
@@ -225,3 +225,9 @@ inputs:
     value:
       acronym: "DoD"
       full_form: "Department of Defense"
+
+  # V-259740
+  - name: org_allowed_postgres_version
+    desc: "Latest allowed Postgres version"

```

Technically this would be the _oldest_ allowed Postgres version, right? The test says the actual installed version should be equal to or later than this one

---

Agreed. I may call it: 

```suggestion
 - name: min_org_allowed_postgres_version
    desc: "The minimum Postgres version allowed by the organization"

```

---

My suggested changes thing didn't work as I wanted in this comment, just FYI

---

renamed to make variable definition clearer